doc/api/deprecations.md

```diff
@@ -2214,6 +2214,22 @@ the `_handle` property of the `Cipher`, `Decipher`, `DiffieHellman`,
 Using the `_handle` property to access the native object is deprecated because
 improper use of the native object can lead to crashing the application.
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: dns.lookup() supports for a falsy hostname
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/23173
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+Previous versions of Node.js supported `dns.lookup()` with a falsy hostname
+like `dns.lookup(false)` due to backward compatibility long before.
```

Comment:
I think we can either replace ` long before.` with ` reasons.` or just remove it altogether as `backward compatibility` should imply that.

Reply:
@lundibundi Thanks for your detailed guidance.